domains/ravost.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Ravost99",
+    "discord": "Ravost99#3361"
```

Comment:
email pls?
if dont have / wanna leak, then leave a empty json object like `{"email":""}`

Reply:
I dont want to share my email is that ok?